### PR TITLE
[IMP] purchase: SCSS revamp 

### PR DESCRIPTION
**Pull request by morm-odoo**
Prior to this PR, the dashboard was a table and there was
unnecessary scss.
This commit changes the table into the new bs5 grids
and clean the scss files.

task-2906527

follows:
- https://github.com/odoo/odoo/pull/95732

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/purchase/static/src/xml/purchase_dashboard.xml [hunk 1]
```diff
@@ -1,68 +1,76 @@
...
-                    <tr>
-                        <td class="o_text">Lead Time to Purchase</td>
-                        <td><span><t t-esc="values['all_avg_days_to_purchase']"/> &#160;Days</span></td>
-                        <td class="o_text">RFQs Sent Last 7 Days</td>
-                        <td><span><t t-esc="values['all_sent_rfqs']"/></span></td>
-                    </tr>
-                </tbody>
-            </table></div>
-        </div></div>
+        <div class="o_purchase_dashboard py-4 border-bottom bg-white">
```
Comment (stefanorigano) [inline addons/purchase/static/src/xml/purchase_dashboard.xml:4]: Too much condensed in `md` screen and too much spread apart in `xxl` :/

`md`
---
image

---------


`xxl`
---
image

---------

I'd suggest something like this
```xml
           <div class="row justify-content-between gap-3 gap-lg-0 px-4">
                <div class="col-12 col-lg-auto col-xxl-5 flex-grow-1 flex-xxl-grow-0 flex-shrink-0">
                    <div class="grid h-100 gap-1 gap-lg-2">
                        [...]
                    </div>
                </div>
                <div class="col-12 col-lg-auto flex-shrink-0">
                    <div class="grid h-100 gap-1 gap-lg-2">
                        [...]
                    </div>
                </div>
            </div>
```
           
What do you think?

**Review by stefanorigano — CHANGES_REQUESTED** [review]

File: addons/purchase/static/src/xml/purchase_dashboard.xml (hunk 1 above)
Comment (morm-odoo) [reply addons/purchase/static/src/xml/purchase_dashboard.xml:4]: Indeed it looks better, I changed it that way :)

**Review by stefanorigano — COMMENTED** [review]
Have you noticed this layout in `md`? I'm not sure👇🏻

image

-----------

Labels seems to take too much room in `xxl`👇🏻
If you can find a workaround good, otherwise we can live with that.

image

**Comment by stefanorigano** [comment]
@morm-odoo , I pushed a commit on top.

**Comment by stefanorigano** [comment]
@morm-odoo , you can remove the [DRAFT] state 👍 
@adr-odoo ,  ready for tech. review

File: addons/purchase/static/src/xml/purchase_dashboard.xml [hunk 2]
```diff
@@ -2,41 +2,53 @@
 <templates>
     <!-- This template is for a table at the top of purchase views that shows some KPIs. -->
     <t t-name="purchase.PurchaseDashboard">
-        <div class="o_purchase_dashboard container">
-        <div class="row">
+        <div class="o_purchase_dashboard align-items-start flex-column flex-lg-row flex-shrink-0 d-flex py-4 border-bottom bg-white">
```
Comment (adr-odoo) [inline addons/purchase/static/src/xml/purchase_dashboard.xml]: container-fluid and you will be able to remove px-3 on table child

**Review by adr-odoo — CHANGES_REQUESTED** [review]
squash wip commit and see container-fluid comment

**Review by stefanorigano — APPROVED** [review]

**Review by adr-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Comment by adr-odoo** [comment]
@robodoo r+

**Comment by ticodoo** [comment]
Hello @morm-odoo ! This PR did nice work, but I was recently alerted to how this change made it so certain languages have cut off text values (German demonstrated in screenshots thanks to its extra long nouns) in v16

image
image

Unfortunately the milk change made it even worse (and makes the issue also apply to Dutch):

image

With the previous table design it was ok-ish because the text overlap was at least in front so it was still legible:
image

I'm not design/bootstrap savvy enough to know if it's possible to fix this with the current `div` design. What do you think?

**Comment by morm-odoo** [comment]
@ticodoo Hey! Thanks for your feedback, it's an issue with the g-col classes for the responsiveness, I'll make a task in rd-design and I'll handle it! :)

**Pull request closed without merging**